actionview/lib/action_view/layouts.rb

```diff
@@ -204,46 +204,15 @@ module Layouts
     include ActionView::Rendering
 
     included do
-      class_attribute :_layout, :_layout_conditions, :instance_accessor => false
-      self._layout = nil
-      self._layout_conditions = {}
-      _write_layout_method
-    end
+      class_attribute :_layout_param, :_layout_conditions, :_layout_root, :instance_accessor => false
+      self._layout_param      = nil
+      self._layout_conditions = nil
+      self._layout_root       = self
 
-    delegate :_layout_conditions, to: :class
+      include LayoutMethod
```

Comment:
There is a reason for this module? I think we can remove it and just change the implementation of `_layout` that we already have on this file.